site/_posts/2023-03-10-growing-pains-attrition.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: blog-post
+title: Growing pains&#58; part I
+description: Building a team is harder than it looks. The attrition problem.
+author: Daniel DeGroff
+category: article
+image: blogs/consents-app/managing-consent-header.png
+tags: startup bootstrap dev hiring firing attrition
+excerpt_separator: "<!--more-->"
+---
+
+Growing is painful, and building an engineering team is hard. There are many reasons for this, and one of them is attrition. In this article I'll talk about my pain and suffering related to attrition while building FusionAuth. 
+
+<!--more-->
+
+### Statistics
+
+The average tenure of a software engineer varies. Interestingly, tenure may be lowest in the largest tech companies such as Google and Amazon.
+
+One report shows an average tenure in big tech is between 1.1 and 3.2 years.[^1] The US Bureau of Labor and Statistics report a nationwide average of 3.8 to 4.3 years which is not limited to the software industry.[^2] Regardless of which statistic you use, it is hard to argue that average tenure in big tech is short, and almost certainly lower than the average across all sectors.
+
+### Bye, Felisha
```

Comment:
```suggestion
### Bye, Felicia
```

Reply:
In the movie Friday, the characters name is `Felisha`, so that is what I was planning to go with. 

https://www.imdb.com/title/tt0113118/fullcredits

Reply:
I even searched for this meme, and all of them said "Felicia"